kura/distrib/src/main/resources/Win32-nn/start_kura_debug.bat

```diff
@@ -0,0 +1,32 @@
+@echo off
+
+set KURA_ROOT=c:\opt\eclipse
+set KURA_HOME=%KURA_ROOT%\kura\kura
+set KURA_HOME_URI=/%KURA_HOME:\=/%
+set KURA_DATA=%KURA_ROOT%\kura\data
+set KURA_TEMP=\tmp\.kura
```

Comment:
Yes and that's actually done by the setup.
Our idea was to keep hard-coded paths in the first instance, so that a user could run Kura without installing - by just copying the files to fixed directory locations (similar to Linux paths).
The NSIS setup replaces all these hard-coded paths with final destinations.